frontend/src/pages/LandingPage.js

```diff
@@ -0,0 +1,12 @@
+import {NavLink} from "react-router-dom";
+
+export default function LandingPage(){
+
+    return(
+        <div>
+            <h2>Hello potato :)</h2>
```

Comment:
what if there is an onion?

Reply:
onions will not be able to use the app sry